lib/foreman/renderer/errors.rb

```diff
@@ -54,6 +58,10 @@ class UndefinedSetting < RenderingError
       class UnsupportedOS < RenderingError
         MESSAGE = N_('Unsupported or no operating system found for this host.').freeze
       end
+
+      class UnknownResource < RenderingError
+        MESSAGE = N_("Unkown '%{klass}' resource class").freeze
```

Comment:
This needs to be single quotes, otherwise it tries to call klass right away (I think that's what the test failures are about)
```suggestion
        MESSAGE = N_('Unkown '%{klass}' resource class').freeze
```

Reply:
This won't work:
```
syntax error, unexpected '}', expecting =>
... MESSAGE = N_('Unkown '%{klass}' resource class').freeze
...                              ^
/home/vagrant/foreman/lib/foreman/renderer/errors.rb:64: syntax error, unexpected ')', expecting `end'
...own '%{klass}' resource class').freeze
...                              ^
 (SyntaxError)
```

Problem is that tests works fine on my setup
```
bundle exec rails t test/unit/foreman/renderer/*_mode_renderer_test.rb
```
but in github it fails with
```
[Foreman::Renderer::Errors::UnknownResource] exception expected, not
Class: <NoMethodError>
Message: <"undefined method `authorized' for UnknownResource:Class">
---Backtrace---
lib/foreman/renderer/scope/macros/inputs.rb:60:in `find_resource'
lib/foreman/renderer/scope/macros/inputs.rb:44:in `input_resource'
```